lib/util/lifted/influx/httpd/handler.go

```diff
@@ -571,8 +571,9 @@ func (h *Handler) AddRoutes(routes ...Route) {
 		}
 
 		handler = h.responseWriter(handler)
-		if r.Gzipped {
+		if r.CompressSupported {
```

Comment:
here, we may need to add a registry for different compressor typeS, I want to plan a refactor item, not in this PR.